### PR TITLE
Feature/individual email update

### PR DESCRIPTION
**Pull request by cthorn2**
Update to the individual email view to add word wrap, hyperlinks directly in the text and updates to the Google Vision to improve text recognition.

### CODE REVIEW

**Review by mdconatser — DISMISSED** [review]

File: lib/ui/mail_view_indv.dart [hunk 1]
```diff
@@ -281,6 +287,9 @@ class MailPieceViewWidgetState extends State<MailPieceViewWidget> {
     } //end element(x) for loop
   } //end _getLinkHtmlFromEmail
 
+
+
```
Comment (AliFahimi) [inline lib/ui/mail_view_indv.dart:207]: Remove these unneeded lines

File: lib/ui/mail_view_indv.dart [hunk 2]
```diff
@@ -322,6 +331,23 @@ class MailPieceViewWidgetState extends State<MailPieceViewWidget> {
...
+  Future<String> _reformatMailPieceString(String x) async {
+    final find = '\n';
+    final replaceWith = ' ';
+    final String original = x;
+    final originalSplit = x.split('\n');
+    for(int i=0; i< originalSplit.length; i++) {
+        if(originalSplit[i].length < 50)
+          originalSplit[i] += '\n';
+        //originalSplit[i] = originalSplit[i].substring(0, originalSplit[i].length - 2);
```
Comment (AliFahimi) [inline lib/ui/mail_view_indv.dart]: If we might not need this later, can you remove these commented out lines?

File: lib/ui/mail_view_indv.dart [hunk 3]
```diff
@@ -339,8 +365,11 @@ class MailPieceViewWidgetState extends State<MailPieceViewWidget> {
         ),
       body: SingleChildScrollView(
         child: Container(
-          padding: EdgeInsets.all(15.0),
+          alignment: Alignment.topCenter,
+          margin: EdgeInsets.all(10.0),
+          //padding: EdgeInsets.all(2.0),
```
Comment (AliFahimi) [inline lib/ui/mail_view_indv.dart]: Remove commented out line

**Review by AliFahimi — CHANGES_REQUESTED** [review]
Looks good to me, just needs a small amount of formatting

File: lib/ui/mail_view_indv.dart (hunk 2 above)
Comment (cthorn2) [reply lib/ui/mail_view_indv.dart]: Removed commented code

File: lib/ui/mail_view_indv.dart (hunk 3 above)
Comment (cthorn2) [reply lib/ui/mail_view_indv.dart]: Removed commented code

File: lib/ui/mail_view_indv.dart (hunk 1 above)
Comment (cthorn2) [reply lib/ui/mail_view_indv.dart:207]: Still needed

**Review by AliFahimi — DISMISSED** [review]

**Review by mdconatser — DISMISSED** [review]

**Comment by cthorn2** [comment]
Here is an image of the changes.
image

**Comment by cthorn2** [comment]
Here is an screen capture of the changes.
image

**Review by shaneknows — DISMISSED** [review]

**Review by mdconatser — DISMISSED** [review]

**Review by shaneknows — APPROVED** [review]

**Review by mdconatser — APPROVED** [review]

**Pull request merged**